2022q2/dpaa2.adoc

```diff
@@ -0,0 +1,39 @@
+=== NXP DPAA2 support
+
+Links +
+link:https://github.com/mcusim/freebsd-src/commits/lx2160acex7-dev[Development] +
```

Comment:
By "Development", do you mean "Change log" or "Change history"? Either seems more natural here, I think.